### PR TITLE
Issue 1294: exclude .tests.info when using updater

### PR DESCRIPTION
**Pull request by herbdool**
Addresses https://github.com/backdrop/backdrop-issues/issues/1294

### CODE REVIEW

File: core/includes/updater.inc [hunk 1]
```diff
@@ -144,6 +144,11 @@ abstract class Updater implements BackdropUpdaterInterface {
       return FALSE;
     }
     foreach ($info_files as $info_file) {
+      // Exclude info files for tests.
+      if (backdrop_substr($info_file->filename, -10) == 'tests.info') {
```
Comment (opi) [inline core/includes/updater.inc]: This would prevent a contrib module named "footests" to be found, isn't it ?

Comment (quicksketch) [reply core/includes/updater.inc]: Good point @opi. I modified it slightly to include leading period and lengthened it to 11 characters.

**Comment by quicksketch** [comment]
Rebuilding sandbox for testing.

**Comment by jenlampton** [comment]
closing sandbox.

**Comment by jenlampton** [comment]
reopening for new sandbox.

**Comment by jenlampton** [comment]
closing for another sandbox.

**Comment by jenlampton** [comment]
one more time. Sandboxme!

**Comment by jenlampton** [comment]
this PR doesn't seem to generate a working sandbox...  testing locally.

**Comment by quicksketch** [comment]
Sandboxes were broken by https://github.com/backdrop/backdrop-issues/issues/3364#issuecomment-461988378. Reopening now that the breaking change is reverted.

**Comment by quicksketch** [comment]
Well after all that I tried this out locally and it didn't introduce any new problems. Without the ability to reproduce this directly, I've merged this anyway because for those affected the code change should help. Merged into 1.x and 1.12.x. Thanks!

**Pull request merged**